tests/test_app_client.py

```diff
@@ -94,6 +94,7 @@
 ROVER_RENTAL_ROBOTS = [ROVER_RENTAL_ROBOT]
 FILTER = "filter"
 ERRORS_ONLY = True
+LOG_LEVELS = []
```

Comment:
Might want to have some random strings in here since the default is also an empty array

Reply:
done!